### DISCUSSION TITLE
Bread Crumb Default for "folders"?

### DISCUSSION THREAD

**Discussion by gihad** [requarks/wiki#3136] [Help / Questions]
Should there be a default page for "folders" that shows up in the breadcrumb?
I'm finding that there are a bunch of broken links for those cases, one example is the `guide` breadcrumb in the official documentation.

Maybe a good default would be a page that shows an index of all pages under that folder? or perhaps the link should be disabled if a page doesn't exist?

This suggestions are aimed at making the wiki content easy to maintain overtime.

**Comment by mattpr** [comment]
I was also wondering same thing.

If I have two pages:
`/folder/foo` and `/folder/bar`
The user sees the `/folder` folder and can click on it to see `/folder/foo` and `/folder/bar` and then can click on them to navigate.

If a user then navigates to `/folder` (via `folder > foo` breadcrumb or manual editing of URL), they get a page not found error.

So I can create a page at `/folder` as a sort of folder-default or folder-root page and it sort of works.  

I was also thinking it would be nice to have a default so when you clicked on a "folder" you would see something like a tree-view of child pages as a default page (if a page hasn't been created at that path...like my `/folder` page example).  However imagine you have a "deep" tree on your wiki and every directory you click on/expand loads a new view...that would be pretty chaotic and potentially slow vs the current model of navigating the tree view by clicking on folders but not loading anything until you click on an actual page.  But perhaps providing a default page (that can be replaced for users that want) with the same behaviour as my `/folder` page in the example would be interesting.  Users would still need to click on the page for it to show up to avoid navigating/loading as each folder is expanded in the tree view.

**One annoying UX thing.**  If I create this folder-root page (`/folder`) and navigate to it... instead of showing the expanded folder view like this:

![image](https://user-images.githubusercontent.com/1196252/109015007-f9149a80-76b4-11eb-87cd-f1b3e1f117b2.png)


It pushes me up a level and shows the `/folder` folder collapsed...so the active page (`/folder`) is not highlighted in the left sidebar.  And if I then want to navigate to child pages (`/folder/*`) I need to click twice (once on `/folder` to expand and then again on the child page).

![image](https://user-images.githubusercontent.com/1196252/109015154-22352b00-76b5-11eb-8a8d-9934f4d8f15c.png)
